src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -193,6 +198,20 @@ public void start(AbstractProject<?, ?> project, boolean newInstance) {
         // We should always start the trigger, and handle cases where we don't run in the run function.
         super.start(project, newInstance);
         
+        try {
+            Map<Integer, GhprbPullRequest> prs = getDescriptor().getPullRequests(project.getFullName());
+            if (prs != null) {
+                prs = new ConcurrentHashMap<Integer, GhprbPullRequest>(prs);
+                if (pullRequests == null) {
+                    pullRequests = prs;
+                } else {
+                    pullRequests.putAll(prs);
```

Comment:
Only open PRs.  The array was kept statically before, and the logic has existed for a while to store the information.  I have been trying to figure out where would be appropriate to keep the map, and thought it would be most appropriate closer to the job.  It would be nice if there were meta data that could be stored in github instead